frontend/lib/queries/autogen/AllSessionInfo.graphql

```diff
@@ -37,6 +37,16 @@ fragment AllSessionInfo on SessionInfo {
     trackingNumber,
     letterSentAt,
     fullyProcessedAt,
+    pdfBase64,
```

Comment:
So I know we talked about the storage costs of keeping these, but we didn't talk as much about the data costs of sending the full object to the frontend (tho obvi that's the whole point). I *think* it's probably still ok given the small number of KB, but we should ask SAJE to try this out on an old phone or use one of these solutions to simulate poor network connectivity. https://www.outsystems.com/blog/posts/simulate-slow-network/. At the very least take a look at the network tab on Chrome DevTools and see a) how much data it's sending and b) how long it takes to load.

One other option here (which, now that I think about it, sounds the best) is to filter out the `pdfBase64` field of the Habitability Letter (i.e. don't send that field to the frontend) and *only* fetch it from the backend when the user clicks `Download PDF`. It will make it a little slower for that user but faster/less data for everyone else. That would involve something like making the `Download PDF` button into a submit button for a form (kind of like how the `edit` buttons on the `Account Settings` page work), adding a mutation in `schema.py` called `FetchLetterPDFMutation` and having that `filter` for just the PDF bytes you're looking for.